fluent-syntax/src/serializer.js

```diff
@@ -207,6 +215,8 @@ function serializeExpression(expr) {
       return serializeCallExpression(expr);
     case "SelectExpression":
       return serializeSelectExpression(expr);
+    case "VariantList":
+      return serializeVariantList(expr);
```

Comment:
This isn't needed. `VariantLists` are not expressions and may not be children of `Pleceables`.